frontend/apps/console/src/features/flows/components/resources/steps/execution/ExecutionMinimal.scss

```
@@ -131,6 +131,16 @@ html[data-color-scheme='dark'] {
     min-width: 200px;
     border: none;
     padding: 24px;
+    // Clicking the body opens the step's properties — signal it.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)